tests/conftest.py

```
@@ -223,36 +223,27 @@ def mw_client(mock_site: MagicMock) -> MwClientPage:
+    with app.app_context():
+        # Create only real tables; skip view-backed mapped classes
+        real_tables = [t for t in _db.metadata.tables.values() if not t.info.get("is_view")]
+        _db.metadata.create_all(_db.engine, tables=real_tables)
+
+        # Create views manually (SQLite-compatible CREATE VIEW)
+        for table in _db.metadata.tables.values():
+            if table.info.get("is_view") and table.info.get("create_query"):
+                try:
+                    _db.session.execute(text(table.info["create_query"]))
+                    _db.session.commit()
+                except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)